js/turtledefs.js

```diff
@@ -116,9 +116,9 @@ function createHelpContent() {
         [_('Delete plugin'), _('Delete a selected plugin.'), 'header-icons/plugins-delete-button.svg'],
         [_('Select language'), _('Select your language preference.'), 'header-icons/language-button.svg'],
         [_('New Project'), _('Initialise a new project.'), 'header-icons/new-project-button.svg'],
-        [_('Restore'), _('Restore blocks from the trash.'), 'header-icons/restore-trash-button.svg'],
+        [_('Undo'), _('Restore blocks from the trash.'), 'header-icons/restore-trash-button.svg'],
```

Comment:
Please don't revert back to Undo. People that that confusing. This restores things from the trash.